melos.yaml

```diff
@@ -9,7 +9,19 @@ ide:
   intellij: false
 
 scripts:
-  prepare: melos bootstrap && dart ./scripts/compile_webworker.dart && dart ./scripts/init_sqlite_wasm.dart && dart ./scripts/init_powersync_core_binary.dart
+  prepare:
+    description: Download SQLite3 WASM for demos
+    run: |
+      melos bootstrap && dart ./scripts/init_sqlite_wasm.dart && dart ./scripts/compile_webworker.dart && dart ./scripts/init_powersync_core_binary.dart
+      for dir2 in demos/*; do
```

Comment:
Will this will work on all operating systems? I think we've had some reported issues of people using this repo on Windows before where some shell scripts did not work. 

Is it not possible to use Melos to execute the `dart run powersync:setup_web no_worker` command on the filtered monorepo folders?

Reply:
As far as I understand it with `melos`, we can filter multiple package directories to run a command in each folder. I don't think it is possible to filter arbitrary directories like the `demos` directories and run a command in each through `melos`

Reply:
Reworked it to run with `melos`.